WooCommerce/Classes/ViewModels/Order Details/Shipping Labels/ShippingLabelPrintingStepView.swift

```diff
@@ -0,0 +1,47 @@
+import SwiftUI
+
+/// Displays one of the steps on how to print a shipping label on an iOS device.
+struct ShippingLabelPrintingStepView: View {
+    struct ViewModel {
+        let stepIndex: Int
+        let text: String
+        let isLastStep: Bool
+    }
+
+    private let viewModel: ViewModel
+
+    init(viewModel: ViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Spacer()
+                .frame(height: 14)
+            HStack(alignment: .top, spacing: 11) {
```

Comment:
Would it make sense to extract this Stack to its own view?

Reply:
Hmm for this one, I'm still debating 🤔 My current logic for whether to create a separate SwiftUI view is its reusability - if it's not reusable by other views (not really like a "component"), it might be easier to read when its UI configuration in one place. Complexity of a view might also be a factor, but this could also be subjective. Curious to hear what the team thinks about when to make a separate SwiftUI view @woocommerce/ios-developers !

Reply:
I don't think reusability should be a concern here. Apple explicitly suggest breaking views down as much as possible, as it is pretty much free, and it helps with readability, which is a concern when it comes to SwiftUI

Reply:
Sure! It was hard for me to tell which views are "complicated" and should be broken down, it's nice you pointed it out for readability 😄 

In https://github.com/woocommerce/woocommerce-ios/pull/3285/commits/37152c4e00de749bf6d97f3d505d4dec90100268, I separate the `HStack` to `ShippingLabelPrintingStepContentView` (lemme know if you have other naming suggestions), and also removed the `ViewModel` per discussion in https://github.com/woocommerce/woocommerce-ios/pull/3322#discussion_r540281521. Also, in https://github.com/woocommerce/woocommerce-ios/pull/3285/commits/7fdcd60e813f1224d04cbc3a8fb686f0b0ac0a53 I renamed the plural steps view to `*StepListView` based on [your feedback](https://github.com/woocommerce/woocommerce-ios/pull/3322#discussion_r542012968) in another PR 🙂